recipe/meta.yaml

```diff
@@ -20,20 +20,25 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - autoconf    # [not win]
-    - automake    # [not win]
-    - libtool     # [not win]
+    - autoconf ==2.71   # [not win]
+    - automake ==1.16.5   # [not win]
+    - libtool  ==2.4.6   # [not win]
     - pkg-config  # [not win]
-    - make        # [not win]
+    - make      # [not win]
     - m2-patch    # [win]
-    - patch       # [not win]
+    - patch ==2.7.6      # [not win]
   host:
-    - icu         # [not win]
-    - libiconv    # [not linux]
-    - xz          # [not win]
-    - zlib
+    # solve icu dependency conflicts
+    - icu ==68.1         # [not win]
```

Comment:
just wondering where these versions that host deps are pinned to come from? These should be older ones if possible, to ensure as much compatibility in the environment as possible, for example, `xz` and `zlib` here are pinned to the newest versions.

Reply:
There were just after searching for available versions in different architectures.

Reply:
This were mainly for testing purposes